### DISCUSSION TITLE
resource creation rate-limit reached

### DISCUSSION THREAD

**Discussion by nrebelski** [argoproj/argo-workflows#7729] [Q&A]
I am facing a problem where I can't increase the number of parallel jobs created from Argo (argo-server and workflow-controller are version v3.2.7). 
I get this error
time="2022-02-02T08:05:42.098Z" level=info msg="Mark node job-t8hfz[0].data-type-loop(0:payables)[0].company-id-loop(3:11216) as Pending, due to: resource creation rate-limit reached" namespace=argo workflow=job-t8hfz

I have a workflow where I have 80 jobs to be done.
My Argo server can't exceed more than 8 simultaneous running jobs. I thought that resourceRateLimit option is responsible for this limit, but I set it to 20. 
Is there something other option to change in order to be able to start more than 8 jobs simultaneously?

Here is a snip from my configmap.yaml
apiVersion: v1
kind: ConfigMap
metadata:
  name: workflow-controller-configmap
data:
  parallelism: "20"
  namespaceParallelism: "20"
  resourceRateLimit: |
    limit: 20
    burst: 2
  artifactRepository: |
.........

**Comment by terrytangyuan** [comment]
By “80 jobs”, do you mean “80 workflows”? How many steps are there in each of the workflows?

I think you probably want to remove other options besides parallelism.

**Reply by nrebelski** [reply]
There is one worflow which consists of 80 jobs. According to the settings the result should be that 20 pods will be created and have running status at the same time. But currently only 8 pods are created, after one pod has finished its job, workflow controller is starting another pod until all 80 jobs will be finished.

**Reply by nrebelski** [reply]
I also set only parallelism setting, but it didn't help.

**Reply by amithi-cider** [reply]
Did you solve the issue? we are experiencing the same issue

**Reply by AniketGroww** [reply]
Hey is it solved, as I am also facing the same issue, is there a way to solve this
